dunstrc

```diff
@@ -111,8 +111,15 @@
     #  * anything else will be interpreted as a X color.
     separator_color = frame
 
-    # Sort messages by urgency.
+    # Sort type.
+    # possible values are:
+    #  * id: sort by id
+    #  * urgency_ascending: sort by urgency (low>normal>critical)
+    #  * urgency_descending: sort by urgency (critical>normal>low)
+    #  * update: sort by update (most recent always at the top)
     sort = yes
+    # Sort fallback ascending, otherwise descending.
```

Comment:
Maybe a better description? What this does is that if two notifications are the same, we sort them by id as a fallback, this controls if this fallback sort is ascending or descending

Reply:
Yeah you could explain it a bit more thorough in here. But is it neccesary to add a new configuration key for this? Isn't there some more elegant solution for this? Something like integrating the ascending value in the `sort` key, since you already specify `urgency_ascending` and -`descending`. Is there a situation you want to specify `urgency_ascending` and `sort_ascending=false`?

Reply:
> Is there a situation you want to specify urgency_ascending and sort_ascending=false

If your notifications are anchored to the bottom-right for example, you might want have the sort_ascending to false, so the most recent appears by id appears at the bottom.

> Isn't there some more elegant solution for this?

I think there is, if we implement the multi-sort I mentioned before, we could have something like

sort = urgency_ascending, update, id_descending

Reply:
I think multi-sort is a little bit complicated as well. Would it be sufficient to base the ID sorting on the anchor (top or bottom)?

Reply:
We can, I'll remove the option then, I struggling to find time to work on this, maybe this weekend

Reply:
Removed the option, now if the origin is set to BOTTOM and the sort type is UPDATE, they appear at the bottom automatically, to preserve the previous behaviour.